static/app/components/dashboards/issueWidgetQueriesForm.tsx

```diff
@@ -83,27 +83,8 @@ class IssueWidgetQueriesForm extends Component<Props, State> {
               widgetQuery={query}
               pageFilters={selection}
               organization={organization}
-              onSearch={field => {
-                // IssueListSearchBar will call handlers for both onSearch and onBlur
```

Comment:
Please double check that the behavior in these components is still as intended and I didn't miss any edge cases 🙏